File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/java/F03.P1.java
class Main {
    static int probe() {
        int x = 41;
        // BEGIN PROBE F03.P1
        x++;
        // END PROBE F03.P1
        return x;
    }

    public static void main(String[] args) {
        System.out.println(probe());
    }
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/zig/F03.P1.zig
const std = @import("std");

fn probe() i32 {
    var x: i32 = 41;
// BEGIN PROBE F03.P1
x += 1;
// END PROBE F03.P1
    return x;
}

pub fn main() !void {
    var threaded: std.Io.Threaded = .init_single_threaded;
    const io = threaded.io();
    var out_buf: [64]u8 = undefined;
    var out = std.Io.File.stdout().writer(io, &out_buf);
    try out.interface.print("{d}\n", .{probe()});
    try out.interface.flush();
}
